### ISSUE TITLE
Super key conflicting now

### ISSUE THREAD

**Issue by ryantm** [CZ-NIC/run-or-raise#79]
Thank you for this excellent extension. I really appreciate it! 

When I recently upgrade to GNOME 46, I found that it isn't working as well as it used to. I use the Super (Windows) key for my run-or-raise shortcuts, and that is also the shortcut that GNOME uses to bring up the mini view of all the windows on your current workspace. Now, run-or-raise sometimes works, and sometimes the mini view comes up and then the run-or-raise shortcut does nothing.

Run or Raise Version 37

**Comment by e3rd** [comment]
Thanks! So unglad to hear about that. What's the default behaviour of the
Super key, without the extension being on? Does it turn on the activities
or so?

**Comment by ryantm** [comment]
I fixed my issue. The problem was that GNOME doesn't like using Right Super as the key that shows all the windows in the workspace, and my keyboard only had a Right super key (Kinesis Advantage2). I had tweaked gnome so Right Super also worked for compose and that it worked for run-or-raise, but it stopped working well with run or raise.

Thankfully, I noticed that my laptop with a Left Super key didn't have this problem, so I remapped my Kinesis Advantage2 to send Left Super when I press the one super key. Now run-or-raise and the workspace activities page works fine.

Once again, thanks for the lovely tool!
